pkg/server/server.go

```diff
@@ -0,0 +1,15 @@
+package server
+
+import (
+	translatev1 "go.expect.digital/translate/pkg/pb/translate/v1"
+	"go.expect.digital/translate/pkg/repo"
+)
+
+type TranslateServiceServer struct {
```

Comment:
Why not just TranslateService?

Reply:
because it has, so logicly this one is ImplementedTranslateServiceServer or shorter `TranslateServiceServer`
`translatev1.UnimplementedTranslateServiceServer`